app/interactors/send_timer_reminders.rb

```diff
@@ -0,0 +1,18 @@
+class SendTimerReminders
+  include Interactor
+  include HasHarvest
+
+  before do
+    context.date ||= Date.current
+  end
+
+  def call
+    User.all.each do |user|
+      next if user.timer_reminder_sent_on.try(:today?)
```

Comment:
:bomb: The problem is that we want to continue past line 11 if the user's _never_ had a timer reminder set, as well.